index.js

```diff
@@ -67,6 +67,54 @@ function toggleColor(element) {
     }
 }
 
+const headerHText = document.querySelector('h1#header')
+
+headerHText.addEventListener('click', () => {
+    toggleColor(headerHText)
+})
+
 /***** Deliverable 2 *****/
+const profileLikeButton = document.querySelector("#profile > button.like-button")
+
+function addLike() {
+    let likeTextLine = document.querySelector("#profile > p.likes")
+    // let numberOfLikes = parseInt((likeTextLine.textContent).split(" ")[0])
+    // let likeWord = (likeTextLine.textContent).split(" ")[1]
+
+    // numberOfLikes ++
+    // newTextLine = `${numberOfLikes} ${likeWord}`
+
+    // likeTextLine.textContent = newTextLine
+    traveler.likes ++
+    likeTextLine.textContent = `${traveler.likes} Likes`
+}
+
```

Comment:
Let's break this one down a bit...

1. find the `p` tag where the current number of likes is stored on the page
2. extract the number out of the string, make sure to convert it to a number with `parseInt()`
3. add 1 to that number.
4. Create a new string with the new number
5. update the `p` tag to have the newly created string as its `innerText`.
```javascript
const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling;
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```
With ES6 notation
```javascript
const likeBtn = document.querySelector(".like-button");
const addNewLike = e => {
    const likesPTag = e.target.previousElementSibling;
    let currentLikes = parseInt(likesPTag.innerText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```